src/gui/torrentcontentmodel.cpp

```diff
@@ -88,6 +99,43 @@ namespace
             return QIcon(iconPixmap);
         }
     };
+#elif defined(Q_OS_MAC)
+    // There is a similar bug on macOS, to be reported to Qt
+    // https://github.com/qbittorrent/qBittorrent/pull/6156#issuecomment-316302615
+    class MacFileIconProvider: public UnifiedFileIconProvider
+    {
+    public:
+        using QFileIconProvider::icon;
```

Comment:
Yes, please. And below too.
Since we have inplace method definitions here we should use rules for definitions (ie. split it with blank line).